storage/tokudb/ha_tokudb.cc

```diff
@@ -7227,10 +7227,28 @@ int ha_tokudb::create(
     const tokudb::sysvars::format_t row_format =
         (tokudb::sysvars::row_format_t)form->s->option_struct->row_format;
 #else
-    const tokudb::sysvars::row_format_t row_format =
-        (create_info->used_fields & HA_CREATE_USED_ROW_FORMAT)
-        ? row_type_to_row_format(create_info->row_type)
-        : tokudb::sysvars::row_format(thd);
+    // TDB-76 : CREATE TABLE ... LIKE ... does not use source row_format on
```

Comment:
I don't understand why a comment describing previous buggy version of this code is required?

Reply:
LAc of history in case it breaks something else? Part of the problem I have with so much of the TokuDB and PerconaFT code and issue trackers is there is no history to understand why things were done the way they were. I suppose the commit message alone might be enough but to me this is exactly the kind of thing that will break something else  later on for some odd reason and there will be no reminder or historic reason documented anywhere why this change was made. The commit message/blame may be enough, assuming that we can keep it forever, but it seemed like a reasonably unclear enough thing to understand the logic and reason why the new code is the way it is to justify a few lines explaining the change.

Reply:
Source repo history has also crossed my mind. Anyway, both versions LGTM if that's OK with you

Reply:
In all fairness, this has been a bug for over 6 years and PZ just found it this weekend, so, how many more times do we expect to have to revisit these 6-8 lines of code :-\